### PR TITLE
docs: update 'Go Cloud' to 'Go Cloud Development Kit' or 'Go CDK' in internal

### PR DESCRIPTION
**Pull request by vangent**
Updates #1186.

### CODE REVIEW

File: internal/contributebot/design/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Go Cloud Contribute Bot Design
+# Go Cloud Development Kit Contribute Bot Design
```
Comment (eliben) [inline internal/contributebot/design/README.md]: Here and below (esp. in the code/config files) - should we just use "Go CDK" for all the internal stuff? It's less typing, and presumably anyone reaching this deep already knows what CDK means.  In particular, "Go Cloud Development Kit Contribute Bot Design" is quite a mouthful :)

Comment (vangent) [reply internal/contributebot/design/README.md]: Done. In some places I just removed it entirely. I did leave the full name in some code files, e.g., for the UI that the running Contribute Bot shows.

File: internal/docs/pubsub/design.md [hunk 2]
```diff
@@ -1,115 +1,132 @@
...
-	http.HandleFunc("/signup", func(w http.ResponseWriter, r *http.Request) {
-		err := t.Send(r.Context(), pubsub.Message{Body: []byte("Someone signed up")})
-		if err != nil {
-			log.Println(err)
-		}
-	})
-	return http.ListenAndServe(":8080", nil)
-}
-```
+func serve() error { ctx := context.Background() client, err :=
```
Comment (eliben) [inline internal/docs/pubsub/design.md]: Oops, what happened here?

File: internal/docs/pubsub/design.md [hunk 3]
```diff
@@ -119,202 +136,175 @@ A more realistic subscriber client would process messages in a loop, like this:
...
+    }
 }
 ```
 
-The messages can be processed concurrently with an [inverted worker pool](https://www.youtube.com/watch?v=5zXAHh5tJqQ&t=26m58s), like this:
-```go
-package main
+The messages can be processed concurrently with an
+[inverted worker pool](https://www.youtube.com/watch?v=5zXAHh5tJqQ&t=26m58s),
+like this: ```go package main
```
Comment (eliben) [inline internal/docs/pubsub/design.md]: Should package main go on a separate line?

File: internal/docs/pubsub/design.md [hunk 4]
```diff
@@ -119,202 +136,175 @@ A more realistic subscriber client would process messages in a loop, like this:
...
 
-import (
-	"context"
-	"log"
-	"os"
-	"os/signal"
+import ( "context" "log" "os" "os/signal"
```
Comment (eliben) [inline internal/docs/pubsub/design.md]: This file also seems to be messed up

File: internal/docs/pubsub/design.md [hunk 5]
```diff
@@ -119,202 +136,175 @@ A more realistic subscriber client would process messages in a loop, like this:
...
+    concrete `pubsub.Topic` object made from it.
+4.  Add `func OpenSubscription(...)` that creates an `acmepubsub.subscription`
+    and returns a `pubsub.Subscription` object made from it.
 
-import (
-	"context"
+Here is a sketch of what the `acmepubsub` package could look like: ```go package
+acmepubsub
+
+import ( "context"
```
Comment (eliben) [inline internal/docs/pubsub/design.md]: Ditto

File: internal/docs/pubsub/design.md (hunk 2 above)
Comment (vangent) [reply internal/docs/pubsub/design.md]: `mdformat` fail. Tried again....

**Comment by vangent** [comment]
> mdformat makes this PR challenging to review :/ Anyway, I searched all appearances of "Go Cloud" here and eyeballed the other changes, seems OK.

Yes, sorry -- I should have done `mdformat` separately.

**Pull request merged**